src/measure_theory/set_integral.lean

```diff
@@ -456,23 +496,63 @@ an `ennreal` number, we use `(μ s).to_real` in the actual statement. -/
 lemma filter.tendsto.integral_sub_linear_is_o_ae
```

Comment:
The docstring of this theorem is now a little bit misleading, because the formal statement includes a function `s (t)`, and also a function `m`. Maybe keep the docstring as it is, but add comments at the end on the role of these choices in the formalization. Same thing for all the next statements.